src/documentation/0009-node-environment-variables/index.md

```diff
@@ -7,7 +7,7 @@ section: Getting Started
 
 The `process` core module of Node.js provides the `env` property which hosts all the environment variables that were set at the moment the process was started.
 
-Here is an example that accesses the NODE_ENV environment variable, which is set to `development` by default.
+Here is an example that accesses the NODE_ENV environment variable, it will return `undefined` untill the variable is set. After every change in `process.env`, you need to restart the server. 
```

Comment:
_until_ is misspelled in the new text. It should probably be reworded to use `if` anyway. 

While we're in here fixing that, let's fix the pre-existing comma splice error.

```suggestion
Here is an example that accesses the NODE_ENV environment variable. It will return `undefined` if the variable is not set. After every change in `process.env`, you need to restart the server. 
```